projects/plugins/jetpack/modules/likes/.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line
```

Comment:
Note: the commit checks were still trying to lint this file, even with it added to the exclude list, which forced me to add this line.

Reply:
The exclude list specifies which files have existing errors that should be ignored, but our tooling still checks *changed* lines. So it doesn't really work to add a new file to the exclude list.

Instead of disabling everything, though, better to disable just the one warning that's being triggered:
```suggestion
// eslint-disable-next-line no-undef
```

Although even better IMO would be to only override what you need to change from the parent config instead of overriding the entire config. Maybe clean up obsolete junk from `projects/plugins/jetpack/modules/.eslintrc.js`, like do we really need to restrict modules to es5 anymore?

Reply:
Good points, @anomiex 👍

I'll give `projects/plugins/jetpack/modules/.eslintrc.js` a cleanup, since even if we still have any ES5 files around, we shouldn't need to check that they're valid ES5 any longer.

Reply:
@anomiex Done. The Jetpack plugin modules lint configuration has been updated to not require ES5 any more.

Reply:
Great, thanks for doing that!